www/js/control/general-settings.js

```diff
@@ -617,4 +629,26 @@ angular.module('emission.main.control',['emission.services',
         });
     }
 
+    var prepopulateQRMessage = {
+        message: $translate.instant('general-settings.qrcode-share-message'), // not supported on some apps (Facebook, Instagram)
+        subject: $translate.instant('general-settings.qrcode-share-subject') // fi. for email
+    }
```

Comment:
Instead of having pre-defined messages here, can we put in the token, or even better, a URL encoding the token? 
Let's start with text.

That would allow people to copy/paste the value or click on a link instead of using a QR code. What if they have only a phone and no computer? They can't use their phone to scan a QR code displayed via email on the same phone.

Reply:
I went ahead and added a `url:` field to the `prepopulateQRMessage` var. I tested out copying and it produces correctly. My emulator can't test texting or emailing, but I believe it will work since it follows the exact same structure as the `share()` function.